.github/ISSUE_TEMPLATE/convergence_epic.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
@JustSlone We were talking about codeowners earlier...I think you should also add yourself (and possibly remove me) as codeowner of `.github`.

Reply:
Ah apparently this line in CODEOWNERS was too general...but we also ought to have separate codeowners for `.github` changes.
```
.git* @ecraig12345
```

Reply:
Sounds good, I'll spin another PR for that

Reply:
https://github.com/microsoft/fluentui/pull/16912